### DISCUSSION TITLE
Do we need ability to lockup native token?

### DISCUSSION THREAD

**Discussion by alexaunter** [galactic3/ft-lockup-contract#3] [Q&A] [answered]
Should we have a way to lockup native token?

If yes, will it be same contract for both FT and NEAR, or should we build two separate contracts?
(new draft implementation implies there's only one token managed per lockup account)

**Comment by AngelBlock** [comment]
Related to #5

**Comment by alexaunter** [comment, marked as answer]
Vlad: No, we don't plan it currently.

**Reply by frol** [reply]
Just to bring some perspective here. I talked to a few people and all immediately said they don't need native NEAR tokens support and even if that is needed, we can get away with wNEAR (wrapped NEAR) FT.

Another concern we had discussed was https://github.com/galactic3/ft-lockup-contract/discussions/5 (I will continue there)
